doc/source/core_developer.md

```diff
@@ -80,6 +80,17 @@ When reviewing, focus on the following:
    tests not only execute the code, but explores corner cases.  It is tempting
    not to review tests, but please do so.
 
+5. **Licensing:** New contributions _must_ be compatible with 
+   {doc}`scikit-image's license <license>`. Which licenses are compatible is a
+   hard question to answer. If in doubt, ask the team for help. If the 
```

Comment:
How about _(...) contributions should be available in the same license or be compatible with (...) question to answer. Examples of BSD-compatible licenses are MIT (link) and Apache (link). If in doubt (...)_

Reply:
+1 to @alexdesiqueira's suggestion; maybe "in the same license" --> "under the same license"